### DISCUSSION TITLE
Etcd failed leader

### DISCUSSION THREAD

**Discussion by Tushar777777** [etcd-io/etcd#16563] [Q&A] [answered]
We have 3 nodes cluster out of which 2 nodes went down and third one shows no leader or failed leader…. How to resolve this
Version 3.5

**Comment by jmhbnz** [comment, marked as answer]
Hey @Tushar777777 - Thanks for your question. Please read and follow the documented etcd `3.5` disaster recovery process here: https://etcd.io/docs/v3.5/op-guide/recovery
